app/views/projects/index.html.erb

```diff
@@ -5,7 +5,10 @@
 <%- else %>
 
 <%- current_user.projects.each do |p| %>
-  <div class="project name"><%= link_to p.name, project_path(p) %>, <span><%= link_to 'see merge requests', project_merge_requests_path(p) %></span></div>
+    <div class="project name">
+        <%= link_to p.name, project_path(p) %>, 
+        <span class="project <%= p.merge_requests.size.zero? ? 'nocounter' : '' %>"><%= link_to merge_request_counter(p.merge_requests.size), project_merge_requests_path(p) %></span>
```

Comment:
Why are you calling a merge_request_helper function on a project view?

Better move the helper function to the project_helper and instead of pass a value as a argument, pass the project itself and the helper will know what to do and return.